concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -45,7 +45,7 @@ def test_upload_should_accept_valid_message(self):
                 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
             ),
             HTTP_AUTHORIZATION = 'Golem ' + encrypted_token,
-            content_type = 'application/x-www-form-urlencoded',
+            content_type = 'multipart/form-data',
```

Comment:
Shouldn't this be added to other tests ? They may pass but we just check `400` response without checking error messages, so there is a chance all of them are returning `400` because of missing this, even if they should return `400` from different reason. Please check.

Reply:
Checked. Everything is ok :)